### ISSUE TITLE
Still "chaos calmer" in 2021?

### ISSUE THREAD

**Issue by pfalcon** [OpenIPC/chaos_calmer#78]
"Chaos Calmer" was a 15.05 (i.e., year 2015) release of OpenWRT: https://openwrt.org/releases/15.05/notes-15.05 . The latest OpenWRT release as of now is 19.07: https://openwrt.org/releases/start.

Is this repository really based on the 2015 fork of OpenWRT? What relation OpenIPC project has with the upstream OpenWRT in general?

(And sorry for dumb questions - but these would be first question a more or less advanced user (i.e. someone who's familiar with OpenWRT) would have. Thanks.)

**Comment by ZigFisher** [comment]
Yes, the current OpenIPC branch, based on OpenWRT code, uses the version from 2015. we also have a second branch that uses Buildroot and has much greater prospects behind it. The project has become independent and is being developed on the GitHub.com/OpenIPC page. My repository is closed for requests, updates, and discussions. Visit our website openipc.org and take part in discussions in telegram groups. Thank you for your interest in the project.

**Comment by ZigFisher** [comment]
As you can see, we have a branch with OpenWRT-19, but there are no people willing to support it. It requires a lot of effort and organization of work. In any case, all three branches have the right to live and are open to discussions and code submissions.

**Comment by pfalcon** [comment]
> As you can see, we have a branch with OpenWRT-19, but there are no people willing to support it.

Actually, I don't see it in this repo, or in other repos in https://github.com/OpenIPC/ org, or even in https://github.com/ZigFisher/chaos_calmer .

Thanks for explaining the situation. I also understand the case that with research/bootstrap work, starting with a stable base is beneficial, rather than bother with regular upstream upgrades. But there's still a concern with longevity of the project, because without updates, it may become "dead weight"/"wasted effort". All in all, I hope it'll come in due time.


> Visit our website openipc.org and take part in discussions in telegram groups.

I'm there actually. I joined out of curiosity to see how much communication happen there. And I see that a lot, and you and other maintainers actively respond there. Which is great and thanks for your time! But my humble suggestion would be also to take some share of that time to add/update more information at openipc.org and github projects/wikis. Because web sites are more accessible to people and can save your from repeated questions ;-).

Thanks for your project, I kept an eye on it for a couple of years and happy to see that it gathers momentum and gives useful deliverables! Would hope to test it and contribute small things, the only concern is available time ;-).

**Comment by ZigFisher** [comment]
Thank you for the feedback ! Yes, we are ready to cooperate and will gratefully accept any help in organizing the project. As so often in other places, there is not enough time for everything. The community's help is invaluable..
